R/approach.R

```diff
@@ -80,12 +80,8 @@ prepare_data <- function(internal, index_features = NULL, ...) {
 
   # Check if the user provided one or several approaches.
   if (length(approach) > 1) {
-    # Figure out which approach we should use for the provided `index_features` (i.e., coalition indices).
```

Comment:
Agree with better/less detailed comment. 
Gave a detailed comment to make it easier for you to understand the logic.